connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectService.scala

```diff
@@ -56,6 +56,8 @@ class SparkConnectService(debug: Boolean)
     extends proto.SparkConnectServiceGrpc.SparkConnectServiceImplBase
     with Logging {
 
+  import SparkConnectService._
```

Comment:
Why do you need this import?

Reply:
To use the `extractErrorMessage` I added in `object SparkConnectService`. I could also import at the top of the file though, do we have a preference here?

Reply:
I assume you can usually do `SparkConnectService.{method_name}` to call the method. However, is there a `extractErrorMessage` in `SparkConnectService`? I cannot find it somehow.

Reply:
oh `extractErrorMessage` is added in this PR :)

Reply:
Right : )

'll remove that import and do the `SparkConnectService. extractErrorMessage`